tests/e2e/app_test.go

```diff
@@ -0,0 +1,52 @@
+package e2e
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("odoAppE2e", func() {
+	// user related constants
+	const loginTestUserApplicationWithoutProject1 = "odoappwithoutprojectuser1"
```

Comment:
Cannot you use ```odoappwithoutprojectuser1, ...``` directly in the scenario itself instead of assigning it to a ```const``` variable and using the same ```const``` variable multiple times.